templates/login/login_core.html.twig

```diff
@@ -1,10 +1,15 @@
-{% set srOnly = (showLabels == true) ? "" : "sr-only" %}
 <!DOCTYPE html>
 <html lang="en">
 <head>
     {% if displayGoogleSignin %}
         <meta name="google-signin-client_id" content="{{ googleSigninClientID|attr }}">
     {% endif %}
+   <script src="https://accounts.google.com/gsi/client" async defer></script>
+    <div id="g_id_onload"
+         data-client_id="{{ googleSigninClientID }}"
```

Comment:
need to html escape this via attr:
```
data-client_id="{{ googleSigninClientID|attr }}"
```

Reply:
Hi, how to do that ? could you give your suggested change? I will add it. Thanks

Reply:
@bradymiller

Reply:
i should of been more clear. you pipe it through the attr to html escape it there:
```
data-client_id="{{ googleSigninClientID|attr }}"
```

Reply:
What does it mean "html escape this via attr" how can I do that? Could you give me sample. thanks

Reply:
here is what the line should look like example (see where the `|attr` is in below line :) ):
```
data-client_id="{{ googleSigninClientID|attr }}"
```

Reply:
Hi Brady,
made changes. please check commit 1d831fc20892c4d33307fd2913df46d5639f5df4 
Thanks